### DISCUSSION TITLE
How do I access the Harman packaged browser?

### DISCUSSION THREAD

**Discussion by GamingAnimate** [airsdk/Adobe-Runtime-Support#594] [Q&A]
Adobe is ending their support for the Flash player on December 31, 2020, and I desperately have to use Flash content in 2021. I have never used this program before, so can somebody please teach me how to?

**Comment by ajwfrost** [comment]
This depends a lot on your scenario and what you're trying to do/who you're trying to support. Can you please submit the contact form on https://services.harman.com/partners/adobe and also if you email adobe.support at harman.com, someone should then get in touch about it.
thanks

**Comment by GamingAnimate** [comment]
I signed in the licence agreement
https://docs.google.com/document/d/1xpQLIRsPP8-7ZvBtbwEeCL7a_JoOVzNaLtcWUCfL2Xg/edit?usp=drivesdk
https://docs.google.com/document/d/12-hWL7uXkegnNjI3TaEsPw0T4wGLsujd7ERSSnkXdOE/edit?usp=drivesdk

**Comment by GamingAnimate** [comment]
I don't think anybody will EVER reply to me. I have a feeling that I'll be left to suffer.

**Comment by yooni126** [comment]
@GamingAnimate I want this packaged browser too, tell me please if you receive any solution...

**Comment by leossmith** [comment]
I believe that the packaged browser is an "enterprise" solution and not meant to be used for some flash game.

**Comment by ajwfrost** [comment]
@GamingAnimate  you appear to have filled in the evaluation contract and information form .. so have you sent that back to who you got it from?

But yes, this is a solution for enterprise customers who need to continue using the Flash Player for business continuity. For end user/personal use the Flash Player end of life isn't really something you can get round (see Microsoft's blog for example for what will be changing on Windows Home Edition computers...)

**Comment by GamingAnimate** [comment]
I am an enterprise customer that needs to access Flash content in 2021. I
sent back the form to whoever gave it to me.

**Comment by ajwfrost** [comment]
@GamingAnimate you'd probably need to work through them then. I couldn't find any emails referencing your surname in our support inbox or in our engineering team's task list. You've omitted a company name from the license contract and put "none" on the information form.

Some of our customers for this are gaming companies but the licensing rules for Flash Player are a little bit more stringent than for AIR..

**Comment by GamingAnimate** [comment]
So that's because I have to enter a company name? Ok, not a problem. I'm
not sure why I said none, but mine is actually Microsoft.

**Comment by GamingAnimate** [comment]
Updated license agreement forms

https://docs.google.com/document/d/13YGbJeYPtOV6-bynkeppRIRg-nK3eWGLLnnqKt0sIBU/edit
https://docs.google.com/document/d/1Xq7-8moETNmMM-9ZTzGZ3gpbXVxiu6vVBWfGvAu67bQ/edit

**Comment by GamingAnimate** [comment]
I'm new to this because I have to use Flash content after eol, and that's why my name wasn't seen.

**Comment by ajwfrost** [comment]
@GamingAnimate I can't access the above forms .. but they need to go back to the Harman sales/account person who gave them to you.
I have to confess, the fact you say that your company is Microsoft is making me doubt that this is a serious enquiry. If it is Microsoft, please contact the browser team and they can put you in touch with the right folk to move it forward..

**Comment by GamingAnimate** [comment]
Don't worry, I'm not pulling any pranks on you, I'm 100% honest with you.
Do they track what you look up?

**Comment by GamingAnimate** [comment]
I'll also send the form back to whoever gave it to me

**Comment by GamingAnimate** [comment]
I sent it to the people that gave me the form

**Comment by dimensionscape** [comment]
> 
> 
> @GamingAnimate I can't access the above forms .. but they need to go back to the Harman sales/account person who gave them to you.
> I have to confess, the fact you say that your company is Microsoft is making me doubt that this is a serious enquiry. If it is Microsoft, please contact the browser team and they can put you in touch with the right folk to move it forward..

@ajwfrost  Ive got a bridge to sell you. It's a really great bridge. A really big bridge. A really cool bridge. 😆
